nuntium/views.py

```diff
@@ -166,7 +166,8 @@ class RejectModerationView(ModerationView):
 
     def get(self, *args, **kwargs):
         get = super(RejectModerationView, self).get(*args, **kwargs)
-        self.get_object().message.delete()
+        self.object.message.public = False
```

Comment:
I think you'll also need to set `self.object.message.moderated = True` here as well. Otherwise the **Moderated** column in the list of messages in the manage section will say ["This message has not been moderated and you can moderate it by clicking here"](https://github.com/ciudadanointeligente/write-it/blob/cdf6ec79c65467f58a337b82154b4249911f1f3c/nuntium/templates/nuntium/profiles/messages_per_instance.html#L81-L93), and when a user clicks it they might inadvertently accept the message.